pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -489,6 +490,12 @@
             + " Using a value of 0, is disabling maxConsumersPerSubscription-limit check.")
     private int maxConsumersPerSubscription = 0;
 
+    @FieldContext(
+        category = CATEGORY_SERVER,
+        doc = "Max size of messages.",
+        maxValue = Integer.MAX_VALUE - InternalConfigurationData.MESSAGE_META_SIZE)
+    private int maxMessageSize = 5 * 1024 * 1024;
```

Comment:
Use constant for default max size. eg: `Commands.DEFAULT_MAX_MESSAGE_SIZE`

Reply:
ok